core/primitives/src/transaction.rs

```diff
@@ -192,6 +197,89 @@ impl SignedTransaction {
         }
         .sign(signer)
     }
+    pub fn send_money(
+        nonce: Nonce,
+        signer_id: AccountId,
+        receiver_id: AccountId,
+        signer: &dyn Signer,
+        deposit: Balance,
+        block_hash: CryptoHash,
+    ) -> Self {
+        Self::from_actions(
+            nonce,
+            signer_id,
+            receiver_id,
+            signer,
+            vec![Action::Transfer(TransferAction { deposit })],
+            block_hash,
+        )
+    }
+
+    pub fn stake(
+        nonce: Nonce,
+        signer_id: AccountId,
+        signer: &dyn Signer,
+        stake: Balance,
+        public_key: PublicKey,
+        block_hash: CryptoHash,
+    ) -> Self {
+        Self::from_actions(
+            nonce,
+            signer_id.clone(),
+            signer_id,
+            signer,
+            vec![Action::Stake(StakeAction { stake, public_key })],
+            block_hash,
+        )
+    }
+
+    pub fn create_account(
+        nonce: Nonce,
+        originator: AccountId,
+        new_account_id: AccountId,
+        amount: Balance,
+        public_key: PublicKey,
+        signer: &dyn Signer,
+        block_hash: CryptoHash,
+    ) -> Self {
+        Self::from_actions(
+            nonce,
+            originator,
+            new_account_id,
+            signer,
+            vec![
+                Action::CreateAccount(CreateAccountAction {}),
+                Action::AddKey(AddKeyAction {
+                    public_key,
+                    access_key: AccessKey { nonce: 0, permission: AccessKeyPermission::FullAccess },
+                }),
+                Action::Transfer(TransferAction { deposit: amount }),
+            ],
+            block_hash,
+        )
+    }
+
+    pub fn delete_account(
+        nonce: Nonce,
+        signer_id: AccountId,
+        receiver_id: AccountId,
+        beneficiary_id: AccountId,
+        signer: &dyn Signer,
+        block_hash: CryptoHash,
+    ) -> Self {
+        Self::from_actions(
+            nonce,
+            signer_id,
+            receiver_id,
+            signer,
+            vec![Action::DeleteAccount(DeleteAccountAction { beneficiary_id })],
+            block_hash,
+        )
+    }
+
+    pub fn empty(block_hash: CryptoHash) -> Self {
+        Self::from_actions(0, "".to_string(), "".to_string(), &EmptySigner {}, vec![], block_hash)
+    }
```

Comment:
These test methods also pollute the public API. The best is to move these methods to testlib, and also introduce the SignedTransactionTesting trait (easy-ext does this effortless).

Apply the same thinking to the rest methods you moved out of `test_utils.rs`

Reply:
see https://github.com/nearprotocol/nearcore/pull/2511#issuecomment-620906318